tests/scripts/build_minimal_appimage_with_custom_apprun.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+if [[ -z ${ARCH+x} ]]; then
+    echo "ARCH not set."
+    exit -1;
+fi
+
+LINUXDEPLOY=$1
+if [[ -z ${LINUXDEPLOY+x} ]]; then
+    echo "LINUXDEPLOY path not set."
+    exit -1;
+fi
+
+RUNNABLE=$2
+if [[ -z ${RUNNABLE+x} ]]; then
+    echo "RUNNABLE path not set."
+    exit -1;
+fi
+
+DESKTOP_FILE=$3
+if [[ -z ${DESKTOP_FILE+x} ]]; then
+    echo "DESKTOP_FILE path not set."
+    exit -1;
+fi
+
+ICON=$4
+if [[ -z ${ICON+x} ]]; then
+    echo "ICON path not set."
+    exit -1;
+fi
+
+
+LINUXDEPLOYDIR=`mktemp -d`
```

Comment:
Creates random unidentifiable temporary directories in the `$TMPDIR`. Adds a lot of unnecessary I/Os. Could use the same pattern as other shell scripts to use a RAM disk if available.

Reply:
There is no trap that cleans up the directory in case of errors. Please use that pattern I designed for test scripts for future scripts...

Reply:
was dropped